src/features/user/Profile/components/RedeemModal.tsx

```diff
@@ -150,105 +156,105 @@ export default function RedeemModal({
-                    })}
-                  </p>
-                  <p className={styles.tempDonationTenant}>
+                <div className={styles.donationCount}>
+                  {t('redeem:myPlantedTreesByOrg', {
+                    count: Number(validCodeData.treeCount),
+                    formattedNumber: getFormattedNumber(
+                      i18n.language,
+                      Number(validCodeData.treeCount)
+                    ),
+                    tpoName: validCodeData.tpos[0].tpoName,
+                  })}
+                  <p className={styles.donationTenant}>
                     {t('donate:plantTreesAtURL', { url: config.tenantURL })}
                   </p>
                 </div>
               </div>
+            </div>
 
-              <div className={styles.shareOptions}>
-                <ShareOptions
-                  treeCount={getFormattedNumber(
-                    i18n.language,
-                    Number(validCodeData.treeCount)
-                  )}
-                  sendRef={sendRef}
-                  handleTextCopiedSnackbarOpen={handleTextCopiedSnackbarOpen}
-                  contactDetails={contactDetails}
-                />
+            {/* hidden div for image download */}
+            <div style={{ width: '0px', height: '0px', overflow: 'hidden' }}>
+              <div className={styles.tempThankYouImage} ref={imageRef}>
+                <div className={styles.tempthankyouImageHeader}>
+                  <p
+                    dangerouslySetInnerHTML={{
+                      __html: t('donate:thankyouHeaderText'),
+                    }}
+                  />
+                </div>
+                <p className={styles.tempDonationCount}>
+                  {t('redeem:myPlantedTreesByOrg', {
+                    count: Number(validCodeData.treeCount),
+                    formattedNumber: getFormattedNumber(
+                      i18n.language,
+                      Number(validCodeData.treeCount)
+                    ),
+                    tpoName: validCodeData.tpos[0].tpoName,
+                  })}
+                </p>
+                <p className={styles.tempDonationTenant}>
+                  {t('donate:plantTreesAtURL', { url: config.tenantURL })}
+                </p>
               </div>
+            </div>
 
-              <Snackbar
-                open={textCopiedsnackbarOpen}
-                autoHideDuration={4000}
-                onClose={handleTextCopiedSnackbarClose}
-              >
+            <div className={styles.shareOptions}>
+              <ShareOptions
+                treeCount={getFormattedNumber(
+                  i18n.language,
+                  Number(validCodeData.treeCount)
+                )}
+                sendRef={sendRef}
+                handleTextCopiedSnackbarOpen={handleTextCopiedSnackbarOpen}
+                contactDetails={contactDetails}
+              />
+            </div>
+
+            <Snackbar
```

Comment:
It doesn't appear like <Snackbar> can be opened, as `textCopiedsnackbarOpen` is always false, and this value is never changed. Should <Snackbar> be removed from RedeemModal